ServerCore/Pages/Teams/Hints.cshtml

```diff
@@ -1,6 +1,48 @@
 ﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/Hints/{puzzleId}"
 @model ServerCore.Pages.Teams.HintsModel
 
+<style>
+    button:disabled {
+        color: #aaaaaa;
+    }
+
+    /*
+     * Strikeout style for hints
+     */
+    .strike1 {
+        position: relative;
+    }
+    .strike1:before {
```

Comment:
I could have sworn I saw and reviewed these before...

Reply:
Indeed, I have them in my copy of the file. What happened? Did this get reverted somehow and you are replacing it?